bitnami/mongodb/values-production.yaml

```diff
@@ -844,6 +844,53 @@ rbac:
   ##
   create: false
 
+## PodSecurityPolicy configuration
+## Be sure to also set rbac.create to true, otherwise Role and RoleBinding
```

Comment:
Do you think it makes sense to add a validation to the `_helpers.tpl` file to throw an error if `podSecurityPolicy.create` is `true` but `rbac.create` is not?

Reply:
Thanks, good call... Again, I wasn't aware of that.

I've updated my branch with that suggestion